layout/js/map.js

```diff
@@ -157,10 +158,12 @@ function hovertext(text, evt){
     stateVal = " ";
   } else {
     var ref = evt.target.getAttribute('xlink:href').split('-')[0];
-    var stateName = ref.replace(/#/g, '')
+    var stateName = ref.replace(/#/g, '');
     var displayNum = Math.round(get_state_value(stateName));
     if (isNaN(displayNum)){
       displayNum = 'no data';
+    } else {
+      displayNum = displayNum.toLocaleString() + ' mgd';
```

Comment:
yeah, I meant within that `else`, but I guess this is what you are doing to add the comma to the numbers? :+1: